dwm1001/examples/lis2dh12.rs

```diff
@@ -1,43 +1,77 @@
 //! Accesses the LIS2DH12 3-axis accelerometer
+//!
+//! To build:
+//! cargo build --example lis2dh12 --features "dev rt semihosting" --release
+//!
+//! To debug:
+//! openocd -f openocd_jlink.cfg
+//! arm-none-eabi-gdb target/thumbv7em-none-eabihf/release/examples/lis2dh12 -x debug.gdb
```

Comment:
Not wrong, but a humble `cargo run --example lis2dh12 --features "dev rt semihosting" --release` should do the same, and doesn't require you to specify this huge path.

Reply:
Indeed. (As I wrote it was just the setup for my system, while experimenting with the debugger plugin in vsode..) I will clean that out. Just a plain `cargo run ... ` is sufficient from prompt.